### DISCUSSION TITLE
Is there any release date for 5.0

### DISCUSSION THREAD

**Discussion by huehnerlady** [ExpediaGroup/graphql-kotlin#1258] [Q&A] [answered]
Hi,

was just wondering, if there is any idea when 5.0 will be released?

Thanks :)

**Comment by smyrick** [comment, marked as answer]
Once I can get the following issues resolved: https://github.com/ExpediaGroup/graphql-kotlin/pull/1259

**Reply by huehnerlady** [reply]
ok thanks
